apps/emqx/include/emqx_lsr.hrl

```diff
@@ -0,0 +1,20 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2025 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+-ifndef(EMQX_LSR_HRL).
+-define(EMQX_LSR_HRL, true).
+
+-define(lsr_err_max_retries, lsr_err_max_retries).
+-define(lsr_err_channel_outdated, lsr_err_channel_outdated).
+-define(lsr_err_restart_takeover, lsr_err_restart_takeover).
+
+-record(lsr_channel, {
+    id :: lsr_session_id(),
+    pid :: pid() | '$1',
+    vsn :: integer() | undefined | '$2'
+}).
+
+-type lsr_session_id() :: emqx_types:clientid().
```

Comment:
Nit: avoid type declaration in headers. It's better to find some module to house the type and export it from there.

Reply:
Yes, also it creates copies of the same type in multiple modules, which may be confusing in dialyzer errors.